plugins/package-managers/node/src/main/kotlin/yarn2/Yarn2.kt

```diff
@@ -67,7 +67,7 @@ data class Yarn2Config(
      * auto-detection can be disabled, and the enabled status of Corepack can be explicitly specified. This is useful to
      * force a specific behavior in some environments.
      */
-    val corepackOverride: Boolean?
+    val corepackEnabled: Boolean?
```

Comment:
@oheger-bosch, does this cause problems with configuration that you'd need to adjust?

Reply:
Yes, this is something we will have to adapt on our side. Thank you for pointing this out.

Reply:
@oheger-bosch, can you please also review otherwise?